src/neuroconv/datainterfaces/ecephys/tdt/tdtdatainterface.py

```diff
@@ -6,7 +8,12 @@ class TdtRecordingInterface(BaseRecordingExtractorInterface):
     """Primary data interface class for converting Tucker-Davis Technologies (TDT) data."""
 
     def __init__(
-        self, folder_path: FolderPathType, stream_id: str = "0", verbose: bool = True, es_key: str = "ElectricalSeries"
+        self,
+        folder_path: FolderPathType,
+        stream_id: str = "0",
+        gain: Optional[float] = None,
```

Comment:
```suggestion
        gain: float,
```

I would be in favor of making this required, since the 'default' is not to be trusted

What do you think?

Reply:
I think until https://github.com/NeuralEnsemble/python-neo/issues/1369 is resolved it is probably the best we can do, the question is whether the gain should be `1.0` in the docs ?

Reply:
Yeah, I'd say the conversion gallery using that value is a good idea for documentations